tests/integration/targets/ntnx_hosts_info/tasks/get_hosts_info.yml

```diff
@@ -0,0 +1,57 @@
+- debug:
+    msg: Start testing ntnx_hosts_info
+- name: test getting all hosts
+  ntnx_hosts_info:
+  register: result
+  ignore_errors: True
+- name: check listing status
+  assert:
+    that:
+      - result.response is defined
+      - result.failed == false
+    fail_msg: "Unable to list all hosts"
+    success_msg: "hosts listed successfully"
+
+- name: test getting particular host using uuid
+  ntnx_hosts_info:
+    host_uuid: '{{ result.response.entities[0].metadata.uuid }}'
+  register: result
+
+- name: check listing status
+  assert:
+    that:
+      - result.response is defined
+      - result.failed == false
+    fail_msg: "Unable to get particular host"
```

Comment:
Please check in assert if the host returned is correct or not. We can use uuid based assert